scripts/create_sfm_from_obj.py

```diff
@@ -0,0 +1,145 @@
+    matcher_conf = match_features.confs['superglue']
+    matcher_conf['model']['weights'] = 'indoor'
+    references = [str(p.relative_to(images)) for p in (images / 'mapping/').iterdir()]
+    print(len(references), "mapping images")
+    
+    # Extract features and matches
+    extract_features.main(feature_conf, images, image_list=references, feature_path=features)
+    pairs_from_exhaustive.main(sfm_pairs, image_list=references)
+    match_features.main(matcher_conf, sfm_pairs, features=features, matches=matches)
+    pairs = sfm_pairs
+    assert features.exists(), features
+    assert pairs.exists(), pairs
+    assert matches.exists(), matches
+
+    # Create a database and import images, features and matches
+    raw_dir.mkdir(parents=True, exist_ok=True)
+    database = raw_dir / 'database.db'
+    create_empty_db(database)
+    camera_mode = 'AUTO'
+    import_images(image_dir=images, 
+                  database_path=database, 
+                  camera_mode=camera_mode, 
+                  image_list=None)
+    image_ids = get_image_ids(database)
+    import_features(image_ids, database, features)
+    import_matches(image_ids, database, pairs, matches,
+                   min_match_score=None, 
+                   skip_geometric_verification=False)
+    geometric_verification(database, pairs, verbose=True)
+
+    # Create images, cameras and points3D binaries
+    write_images_binary(colmap_images, raw_dir / 'images.bin')
+    write_cameras_binary(colmap_cameras, raw_dir / 'cameras.bin')
+    write_points3D_binary({}, raw_dir / 'points3D.bin') #Should be empty
+
+    # Triangulate for 3D points
+    reference = pycolmap.Reconstruction(raw_dir)
+    sfm_dir.mkdir(parents=True, exist_ok=True)
+    database = sfm_dir / 'database.db'
+    image_ids2 = create_db_from_model(reference, database)
+    triangulation.main(sfm_dir, raw_dir, images, 
+                       pairs, features, matches,
+                       skip_geometric_verification=False, 
+                       min_match_score=None, verbose=True)
+    return
+
+if __name__ == '__main__':
+    object_name = os.environ['OBJECT']
+    default_output_path = Path(os.environ['PIXTRACK_OUTPUTS']) / 'nerf_sfm' / object_name
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--mesh_path', required=True, type=Path)
+    parser.add_argument('--output_path', default=default_output_path, type=Path)
+    parser.add_argument('--dataset_path', default=default_output_path, type=Path)
+    args = parser.parse_args()
+
+    #TODO @prajwal: Choose these parameters automatically instead on hardcoding them
+    fx = 900. * 3
+    fy = 900. * 3
+    cx = 512. * 3
+    cy = 512. * 3
+    W = 1024 * 3
+    H = 1024 * 3
+    dataset_path = args.dataset_path
+    output_path = args.output_path
+    mesh_path = args.mesh_path
+
+    # Render images from obj, get colmap images and cameras
+    render_dataset_and_get_colmap_images_and_cameras(mesh_path, dataset_path, 
```

Comment:
return not captured to pass to the next func

Reply:
Fixed